docs/set_up_transfer_and_export_device.rst

```diff
@@ -0,0 +1,210 @@
+   being the most notable exceptions).
+
+   If and when you do need to copy decrypted files in electronic form, the
+   recommendations below are intended to establish a baseline of security.
+   Please consider these recommendations in the context of your own threat
+   model, and do not hesitate to contact us via securedrop@freedom.press
+   (`GPG encrypted <https://securedrop.org/sites/default/files/fpf-email.asc>`__)
+   if we can help.
+
+Choose media types and encryption
+---------------------------------
+You will need to decide what storage media to use for the *Transfer Device* and
+the *Export Device*, and which encryption scheme to apply to each device. There
+are many options to consider: USB flash drives, write-once media like CD-Rs and
+DVD-Rs, external hard drives, and so on.
+
+The following recommendation is intended to balance security, usability and cost
+considerations, and you may want to modify it based on your threat model:
+
+- Use USB flash drives for both the *Transfer Device* and the *Export Device*.
+
+- Encrypt the *Transfer Device* using LUKS, which works in the Tails environment
+  and in other Linux environments.
+
+- Encrypt the *Export Device* using VeraCrypt, which works across platforms.
+
+- Optionally, purchase a hardware USB `write blocker <https://www.forensicswiki.org/wiki/Write_Blockers>`__
+  as used in forensics, and enforce its usage whenever the *Export Device* is
+  attached to an Internet-connected workstation. This ensures that malware
+  cannot spread from infected computers in your network to the *Secure Viewing
+  Station*.
+
+If you follow this recommendation, it is important that the contents of the
+*Transfer Device* and the *Export Device* are always wiped after a copy operation
+is completed.
+
+Write-once media like CDs and DVDs can be a reasonable alternative to this
+setup. If you implement a workflow based on CDs or DVDs, it is crucial that they
+are destroyed immediately after use. While you can find CD/DVD destroyers at a
+relatively low cost, those built to the highest standards of security sell in
+the $2,500 to $3,000 price range as of 2019.
+
+Decide how to manage encryption passphrases
+-------------------------------------------
+Because files are copied between multiple computers, KeePassX in Tails is not
+necessarily the most convenient option for managing the encryption passphrases for
+your *Transfer Device* or your *Export Device*. While Tails itself gives you the
+option to "remember" passphrases, this option does not work across reboots.
+
+A simple alternative is to make sure that every journalist stores the
+*Transfer Device* and *Export Device* passphrases in their own password manager,
+which ideally will synchronize to their smartphone. See the Freedom of the Press
+Foundation guide for `choosing a password manager <https://freedom.press/training/blog/choosing-password-manager/>`__
+if you are not currently using one.
```

Comment:
As above, the guidance on choosing a password manager also belongs in the "Passphrase Best Practices" guide, in an adjacent doc.

Reply:
Added!